pkg/passkey/passkey.go

```diff
@@ -14,17 +14,17 @@ import (
 	"fmt"
 )
 
-// Flags for the credential parameters: https://www.w3.org/TR/webauthn-2/#flags
+// CredentialFlags for the credential parameters: https://www.w3.org/TR/webauthn-2/#flags
 type CredentialFlags struct {
 	UserPresent     bool
 	UserVerified    bool
 	AttestationData bool
 	ExtensionData   bool
 }
 
-// Structure to store information and key of public key credential.
+// Credential is a structure to store information and key of public key credential.
 type Credential struct {
-	Id        string
+	ID        string
```

Comment:
That's a breaking changes in our public API that might warrant at least minor version bump

Reply:
but tbh it's unused: https://pkg.go.dev/github.com/gopasspw/gopass@v1.15.15/pkg/passkey?tab=importedby 
So it's fine.

Reply:
well, fine... except for all of the private packages using it! :P

IMO, _any_ breaking change should necessitate a major version bump. mildly irritating for a case like this, but this change was not made for formatting reasons (which was why this tree existed), so it could have simply been reverted.

Reply:
We did have this discussion when this project was initially published and we have documented our stance in the FAQ: https://github.com/gopasspw/gopass/blob/master/docs/faq.md#api-stability

Reply:
makes sense to me! i wasn't aware of the documented policy, and this change didn't impact me -- i was simply commenting on it to highlight that it _could_ have broken things for someone who doesn't explicitly version their usage of this.